### PR TITLE
editorial: grouped payment details dictionaries into section

### LINKED ISSUE #427: Group payment dictionaries into a section
It might be nice to nest the three payment dictionaries sections under a single section (e.g. "Payment details dictionaries")

### PR DESCRIPTION
**Pull request by kayakalan**
closes #427

***
[Preview](https://s3.amazonaws.com/pr-preview/kayakalan/browser-payment-api/427_sections.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/browser-payment-api/b68348f...kayakalan:dba6777.html)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,14 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta name="generator" content=
```
Comment (marcos-travel) [inline index.html:4]: Need to remove this.

**Review by marcos-travel — CHANGES_REQUESTED** [review]

**Comment by marcoscaceres** [comment]
Marked as non-substantive for IPR from ash-nazg.

**Comment by marcoscaceres** [comment]
Going to merge this, and then remove the tidy mark locally. The indentation risk merge conflicts if this sits for too long.

**Comment by kayakalan** [comment]
Sorry to be slacker. High in the queue to get better and quicker at this.
Thx for the ping.

**Comment by marcoscaceres** [comment]
@kayakalan, no probs. I already took care of it. Thanks for helping out with this. If you want to grab a few more editorial ones, go for it. Again, happy to provide guidance as needed.

**Pull request merged**